### DISCUSSION TITLE
Vercel Docs Typo

### DISCUSSION THREAD

**Discussion by stahlgazer** [vercel/community#3941] [Bug Report]
### Page to Investigate

https://nextjs.org/learn/foundations/how-nextjs-works/development-and-production

### Steps to Reproduce

I believe the docs have a typo that should be changed to an 'as' before the 'the'.

Original: "It comes with features that aim to improve the Developer Experience such the TypeScript and ESLint integration, Fast Refresh, and more."

Fixed: "It comes with features that aim to improve the Developer Experience such **as** the TypeScript and ESLint integration, Fast Refresh, and more."

**Comment by amyegan** [comment]
Hey @stahlgazer. Thanks for reporting this!

**Reply by amyegan** [reply]
It's fixed now
